deploy/before.sql

```diff
@@ -0,0 +1,3 @@
+BEGIN;
+    ALTER TABLE payments_for_open_source RENAME TO investments_in_open_source;
```

Comment:
@whit537 I understand that you're experimenting here, but aren't there quicker/cheaper/safer ways to achieve this than having to rename this throughout the codebase? 😕

Reply:
My suggestion would be to only tinker with copy. Modeling investments as `payments` in code is forgivable, IMO - not worth spending your time changing

Reply:
Fair enough. I've pushed 49c96e72efd71de6a362846cf695dd73f6cba5ba off, we can come back to it later if @JessaWitzel wants to. :D

Reply:
Agree with @rohitpaulk.